src/applications/facility-locator/utils/helpers.js

```diff
@@ -216,3 +219,39 @@ export const isVADomain = website => {
   const regExp1 = /https?:\/\/(?:www\.|staging\.)?va\.gov(\/*)/;
   return regExp1.test(website);
 };
+
+const headingStyle = {
+  fontWeight: '700',
+  fontFamily: 'Bitter, Georgia, Cambria, Times New Roman, Times, serif',
+  lineHeight: '1.3',
+  clear: 'both',
+};
+
+const ddStyle = {
+  margin: '2rem 0 .5rem 0',
+  lineHeight: '1.5',
+};
+
+/**
+ * Helper method to display an urgent care alert dialog
+ *
+ * @param {object} state currentQuery
+ */
+export const showDialogUrgCare = currentQuery => {
+  if (
+    (currentQuery.facilityType === LocationType.URGENT_CARE &&
+      currentQuery.serviceType === 'NonVAUrgentCare') ||
+    currentQuery.facilityType === LocationType.URGENT_CARE_FARMACIES
+  ) {
+    return <UrgentCareAlert ddStyle={ddStyle} headingStyle={headingStyle} />;
+  }
+
+  if (
+    currentQuery.facilityType === LocationType.CC_PROVIDER &&
+    currentQuery.serviceType === CLINIC_URGENTCARE_SERVICE
+  ) {
+    return <UrgentCareAlert ddStyle={ddStyle} headingStyle={headingStyle} />;
```

Comment:
How come `ddStyle` and `headingStyle` are passed here as props, as opposed to just being directly in the `UrgentCareAlert` component?

These could also be class names really easily - 

[fontWeight](https://design.va.gov/utilities/font-weight) -> `.vads-u-font-weight--bold`
[fontFamily](https://design.va.gov/utilities/font-family) -> `.vads-u-font-family--serif`
[lineHeight](https://design.va.gov/utilities/line-height) -> `.vads-u-line-height--3`